sentry_dsn_data.go

```diff
@@ -7,11 +7,21 @@ import (
 	"sync"
 
 	"github.com/getsentry/sentry-go"
+	v1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var DSNAnnotation = "k8s.sentry.io/dsn"
 
+const (
+	POD        string = "Pod"
```

Comment:
Here and below
```suggestion
	KindPod        string = "Pod"
```

Also let's move these to a file with a better name (e.g. `constants.go`)